netbsd/vm/README.md

```diff
@@ -0,0 +1,32 @@
+# NetBSD VM management
+
+## Overview
+
+This is your working directory for auto-installation and using 
+the VM.
+
+
+If it doesn't exist, `disk.qcow2` (the 32G hard-drive of the VM) will be created by `setup.sh` script
+
+If it does, the script will delete it and create again (with user confirmation)
+
+### Prerequisites
+
+These files need to be added to this directory (copied or 
+symlinked).
+
```

Comment:
@Silentsky0 where we can find them? please add some links or info

Reply:
added some information about release links and producing them from local sources